src/applications/claims-status/components/ClaimsDecision.jsx

```diff
@@ -10,8 +10,8 @@ const headerText = closedDate =>
 
 const ClaimsDecision = ({ completedDate, showClaimLettersLink }) => (
   <>
-    <va-alert>
-      <h3 className="claims-alert-header vads-u-font-size--h4" slot="headline">
+    <va-alert usdws>
```

Comment:
We shouldn't be using the `uswds` flag here yet. We've been given guidance to not mix and match the v3 design with the non-v3 design in the same application